### DISCUSSION TITLE
Weekly Meetings

### DISCUSSION THREAD

**Discussion by DixonDalmeida** [ossf/wg-security-tooling#1] [General] [answered]
I would like to be part of the weekly meetings. Can someone let me know how can I add myself to the weekly meeting?

**Comment by psiinon** [comment, marked as answer]
Hiya - I've updated https://github.com/ossf/wg-security-tooling/blob/main/README.md with a link to the Google Meet and the times, but if you email me at simon.bennetts@owasp.org then I'll add you to the meeting invite as well :)

**Comment by fdegir** [comment]
@psiinon I'm interested in this as well. Is it ok I send mail to you to be added to the meeting too?

Ps. PR, fixing google meet link: https://github.com/ossf/wg-security-tooling/pull/2

**Reply by psiinon** [reply]
Of course - that goes for everyone! And thanks for the PR - merged :)
